channels/facebook/utility.js

```diff
@@ -103,19 +103,18 @@ function receivedAuthentication(event) {
  */
 
 function receivedDeliveryConfirmation(event) {
-	var senderID = event.sender.id;
-	var recipientID = event.recipient.id;
-	var delivery = event.delivery;
-	var messageIDs = delivery.mids;
-	var watermark = delivery.watermark;
-	var sequenceNumber = delivery.seq;
+	//var senderID = event.sender.id
```

Comment:
remove comments

Reply:
I left them there on purpose in case we would need them in the future and wouldn't know which parameters we can use